CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 3.0.0 - UNRELEASED
+
+- Documents and prioritizes the existing `apos` command alias as the CLI
+command.
+- More colors! Command logs are no longer all an ❗️alarming❗️ shade of red. Red now means errors.
+
+### Breaks
+
+- The Apostrophe CLI no longer supports running core and Apostrophe module tasks.
+  - The CLI tool was providing an alias for these `apos apostrophe-users:add admin admin`. These can all still be run more directly on the command line with `node app`, e.g., `node app apostrophe-users:add admin admin`.
```

Comment:
Hmmm, not sure if this is the right developer UX choice or not. Shouldn't we be impressing on devs that the CLI is the way to do CLI things with Apostrophe generally? But if this is a settled team decision I'll homer-hedge-disappear

Reply:
Maybe `apos task` so it's unambiguous? Can't hurt to provide that?

Reply:
Our intention is to create clearer CLI tasks for common core tasks. That could include `apos adduser abea admin`, `apos build`. The hard part of using core tasks is remembering them, not the `node app` part. This is the first step of rethinking this feature that I'm not sure anyone really uses.

Reply:
I would think that any aliasing feature to bypass task namespacing would make the most sense in apostrophe itself, so you can register the alias somewhere as you write the task.

Reply:
I think manually typing `node app` for stuff like this feels "this did not get thought through and shipped as a command for me" to folks used to how most node CLI tools behave.

Reply:
I think doing something for this would be good. I don't think this should be it and don't know anyone who knows this is an option and uses it. We could come up with something better. I like the alias option you suggested there.